docs/releases/1.10.1.txt

```diff
@@ -61,3 +61,7 @@ Bugfixes
 
 * Fixed ``contrib.admindocs`` crash when a view is in a class, such as some of
   the admin views (:ticket:`27018`).
+
+* Fixed the ``migrate`` command to include already applied migrations state
```

Comment:
I could but failure to create content types was really just a symptom, for example permissions weren't created either.

Maybe I could simply mention the ``apps`` kwarg passed to migration signal receivers now includes applied migrations?